fuzz/remove-main.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
```

Comment:
Is this the best way to do this? Is there a possible approach that's including libraries directly rather than accessing them through the project's main?

Reply:
So this is a single file project, with just a main.c. There's not really any libraries to include. I'm open to try a different way but I don't have any other ideas to fuzz this.

Reply:
The other approach would be to just make a static copy of the main and replace with libFuzzer, but it's subject to similar issues. 

Looking more closely, I'm not sure I'll be able to justify accepting this since it's just a pet project anyways. However if this were another project, I would be open to two approaches:

1. Easiest - I'll accept as is and we hope to not see too much activity upstream.
2. Harder, but probably better practice - Fork upstream, and modularize the main.c so it serves largely as an entrypoint, and then create an MR upstream (don't add fuzzing yet). Then, if and when upstream accepts the modularization MR, create a new fork from upstream to add fuzzing (and basically add a libFuzzer entrypoint). At this point you're essentially one of the  maintainers of the project so I don't know how feasible this is, but it is technically better practice than a script that modifies the main.c on the fly.

Reply:
Alternative idea - have you tried using libFuzzer as a library?
https://llvm.org/docs/LibFuzzer.html#id40